renderer.php

```diff
@@ -144,7 +144,8 @@ protected function render_digitala_report(digitala_report $report) {
             } else {
                 $reporttitle = 'report-title';
             }
-            $out .= create_card($reporttitle, html_writer::tag('p', get_string('reportinformation', 'digitala')).
+            $out .= create_card($reporttitle, html_writer::tag('p', timestampformatter($attempt->timemodified)).
```

Comment:
![image](https://user-images.githubusercontent.com/6292793/165568167-0405e175-9a61-42e5-bc1d-a13572dce15f.png)
Could this have some kind of definition what this date is? Now it is a little unclear for user to know what this means. Something like `Submitted: 1.1.1970 00.00:00`?

Reply:
Good idea.